recipes/portmidi/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "2.0.3" %}
+
+package:
+  name: portmidi
+  version: {{ version }}
+
+source:
+  url: https://github.com/PortMidi/portmidi/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 934f80e1b09762664d995e7ab5a9932033bc70639e8ceabead817183a54c60d0
+
+build:
+  number: 0
```

Please add a run-export; I cannot [find](https://abi-laboratory.pro/?view=timeline&l=portmidi) it in the abi lab, but based on the name `$PREFIX/lib/libportmidi.so.2`, I think major should be fine
```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage("portmidi", max_pin="x") }}
```